dcli/template/kotlin/src/main/kotlin/app/Solve.kt

```diff
@@ -0,0 +1,46 @@
+package app;
+
+import template.{{ targetFile }};
+
+/**
+ * Class executed when user click on `Submit` button.
+ * It must test if the user has reach or not the solution
+ * with many tests.
+ * Also have to exit(1) if something is wrong
+ * And exit(0) if everything was fine.
+ */
+class Solve {
+    companion object {
+        fun main() {
+            try {
+                {%if targetMethodReturn == "" %}
+                // call both method, user and success, do your own test to compare user code to the success one
+                {{ targetFile }}.{{ targetMethod }}({% if targetMethodArgs != "" %}/* //TODO fill it with your own args */{% endif %});
+                success.{{ targetFile }}.{{ targetMethod }}({% if targetMethodArgs != "" %}/* //TODO fill it with your own args */{% endif %});
+                //TODO call Logger.logSuccess(); if everything is fine
+                //TODO call System.exit() is something went wrong and log it with Logger.logFail().
```

Comment:
In Kotlin favorable to use https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.system/exit-process.html instead of `System.exit()`

Reply:
Done

Reply:
still present

Reply:
Done